crates/serpen/tests/fixtures/ast_rewriting/alias_transformation_test/main.py

```
@@ -0,0 +1,64 @@
+from utils.data_processor import format_output as format_a
+from utils.config_manager import load_config as config_a
+
+# These non-aliased imports should remain unchanged
+import math
+import hashlib
+
+# Mixed from-import: some aliased, some not (partial filtering test)
+from utils.helpers import helper_func, debug_print as debug_a
+
+
+def main():
```

Ruff (0.11.9)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)